### PR TITLE
GP 64 fe list game

### PR DESCRIPTION
**Pull request by dg-panh**
**New Feature:**

- Game List Page
- Remove unused file/folder
- Add debug file

### CODE REVIEW

File: .vscode/setting.json [hunk 1]
```diff
@@ -1,13 +0,0 @@
-{
-    "editor.renderWhitespace": "boundary",
```
Comment (danielNg91) [inline .vscode/setting.json]: uncommit this file

File: package.json [hunk 2]
```diff
@@ -18,9 +18,11 @@
     "@mui/material": "~5.14.10",
     "@mui/styled-engine-sc": "~5.14.10",
     "@reduxjs/toolkit": "~1.9.5",
+    "@types/numeral": "~2.0.3",
```
Comment (danielNg91) [inline package.json]: @types should in devDeps

File: src/Layout/DashboardLayout/Sidebar/Items.tsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
+import { PageName, convertNameToPath } from '../../../common'
 import SvgColor from '../../../components/Svg-color'
 
 
 const icon = (name: string) => <SvgColor src={`/assets/icons/navbar/${name}.svg`} sx={{ width: 1, height: 1 }} />
 
-const sidebarItems = {
+export const sidebarItems = {
```
Comment (danielNg91) [inline src/Layout/DashboardLayout/Sidebar/Items.tsx]: add types for this sidebarItems

File: src/Layout/DashboardLayout/Sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -39,11 +39,11 @@ const renderContent = (
 		<Box sx={{ mb: 5, mx: 2.5 }}>
 			<Link underline="none">
 				<StyledAccount>
-					<Avatar src={account.photoURL} alt="photoURL" />
+					<Avatar src={account.avatar} alt="photoURL" />
 
 					<Box sx={{ ml: 2 }}>
 						<Typography variant="subtitle2" sx={{ color: 'text.primary' }}>
-							{account.displayName}
+							{account.firstName + ' ' + account.lastName}
```
Comment (danielNg91) [inline src/Layout/DashboardLayout/Sidebar/Sidebar.tsx]: template literals for better readablitity

File: src/components/NavSession/NavSection.tsx [hunk 5]
```diff
@@ -14,35 +14,37 @@ type NavItemProps = {
 
 type Props = {
 	data: Array<NavItemProps>,
+	sx?: object,
```
Comment (danielNg91) [inline src/components/NavSession/NavSection.tsx]: 'The `Object` type actually means "any non-nullish value", so it is marginally better than `unknown`.',
        '- If you want a type meaning "any object", you probably want `Record<string, unknown> instead.`',
        '- If you want a type meaning "any value", you probably want `unknown` instead.',

**Review by danielNg91 — CHANGES_REQUESTED** [review]

File: .vscode/setting.json (hunk 1 above)
Comment (dg-panh) [reply .vscode/setting.json]: ủa xóa r mà @@

**Review by danielNg91 — APPROVED** [review]

**Pull request merged**